src/myapp.h

```diff
@@ -146,7 +146,7 @@ struct MyApp : wxApp {
 
         auto serr = ScriptInit(frame->GetDataPath("scripts/"));
         if (!serr.empty()) {
-            wxLogFatalError(L"Script system could not initialize: %s", serr);
+            wxLogFatalError(wxString::Format(_(L"Script system could not initialize: %s"), serr));

```

Comment:
Let's not include this file, errors from here would be highly technical errors that pretty much never happen, they don't need to be localized.
Generally, only things in the graphical UI should be localized, not logging messages.